src/lwc/utilPageLevelMessage/utilPageLevelMessage.css

```diff
@@ -2,6 +2,7 @@
     margin: 0;
     padding: var(--lwc-spacingMedium, 1rem);
     width: 100%;
+    min-width: 10rem !important;
```

Comment:
@psadie-sfdo , @kenlewis-sfdo , @vmartinov-sforg: FYI -- I've added the min-width so the page message can be displayed within the component width when the component is added into the left or right sidebar. Min default is 30rem. I could not set the CSS outside of the page level message child component.

Reply:
Thanks Radmila! This seems fine to me but would love confirmation from @psadie-sfdo since he's more aware of our use of this component.

Reply:
No concerns on setting the mid-width there. Sorry for the delay!